nodestream/pipeline/extractors/queues/extractor.py

```diff
@@ -0,0 +1,88 @@
+import json
```

Comment:
Maybe overkill, but could be beneficial to add some design diagrams to the developer reference: https://nodestream-proj.github.io/docs/docs/category/developer-reference/